src/components/common/buttons/ShareButton.js

```diff
@@ -13,21 +14,25 @@ type ShareButtonProps = {
   buttonProps: any,
 }
 
+const log = logger.child({ from: 'ShareButton' })
+
 const ShareButton = ({ share, onPressDone, actionText, ...buttonProps }: ShareButtonProps) => {
   const [showErrorDialog] = useErrorDialog()
+  const { canShare } = useNativeSharing()
+
+  log.info('getPaymentLink', { share })
 
-  console.info('getPaymentLink', { share })
-  const shareAction = async () => {
+  const shareAction = useCallback(async () => {
     try {
-      await Share.share(share)
+      await Share.share(share).catch(e => alert('qqq', e))
```

Comment:
alert should be removed.
there's an console log for debug reasons (of course is better to wrap it with function check for dev mode)
look for lib/utils/devMode

Reply:
@serdiukov-o-nordwhale 

I pushed this code by mistake - I used it for testing purposes.
Removed in new commit.